examples/fluent-bit/ecs-log-deletion/README.md

```diff
@@ -0,0 +1,135 @@
+
+## Scenario
+
+For this example, consider that your app produces log files with paths like `/var/log/service-{timestamp}.log`. Each hour, your app write to a log file with a new timestamp. Thus, if your app runs for a very long time, you will quickly have lots of log files sitting around on disk. This is not ideal for two reasons:
+1. Your disk may eventually fill up.
+2. The large number of log files may slow down Fluent Bit. Fluent Bit continuously scans for files matched by your configured `Path` and runs a syscall on them to check if they were modified. If you have tons of log files that it must scan through, this will cause it to slow down, and in extreme cases, you may even lose logs. 
+
+Fluent Bit will use this base configuration (based on the ECS Log Collection example) to tail the log files:
+
+```
+[INPUT]
+    Name              tail
+    Tag               service-log
+    Path              /var/log/service*.log
+    DB                /var/log/flb_service.db
+    DB.locking        true
+    Skip_Long_Lines   On
+    Refresh_Interval  10
+    Rotate_Wait       30
+# Output for stdout logs
+# Change the match if your container is not named 'app'
+[OUTPUT]
+    Name cloudwatch
+    Match   app-firelens*
+    region us-east-1
+    log_group_name firelens-tutorial-$(ecs_cluster)
+    log_stream_name /logs/app/$(ec2_instance_id)-$(ecs_task_id)
+    auto_create_group true
+    retry_limit 2
+# Output for the file logs
+[OUTPUT]
+    Name cloudwatch
+    Match   service-log
+    region us-east-1
+    log_group_name firelens-tutorial-$(ecs_cluster)
+    log_stream_name /logs/service/$(ec2_instance_id)-$(ecs_task_id)
+    auto_create_group true
+    retry_limit 2
+```
+
+## Log Deletion Options
+
+### Background
+
+Make sure you are familiar with the [Fluent Bit documentation](https://docs.fluentbit.io/manual/) and with [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) and its container image distribution.
+
+### 1. Use Fluent Bit Exec Input to run deletion command
+
+##### Prerequisites:
+* Follow this [Set up Logging for CDK Applications](https://builderhub.corp.amazon.com/docs/native-aws/developer-guide/cdk-logging.html) to set up Fluent Bit
+* The volume mount for Fluent Bit must not be read-only
+* Familiar with [exec input](https://docs.fluentbit.io/manual/pipeline/inputs/exec) of Fluent Bit
+
+You can run a command that deletes log files by adding an exec input to the configuration. Here is the example:
+
+```
+[INPUT]
+    Name          exec
+    Tag           delete-old-files
+    Command       find /var/log/  -type f -mmin +10080 -execdir  rm -f -- '{}' ';'
+    Interval_Sec  600
+    Interval_NSec 0
+    Buf_Size      8mb
+    Oneshot       false
+
+[OUTPUT]
+    Name          null
+    Match         delete-old-files
+```
+Here is a breakdown of the individual components of the command:
```

Comment:
also explain the FLB config items. Like the `Interval_Sec` to run the command, and the null output for any output from the command. This is because exec takes the output of commands and turns it into a log.